adapter/src/ethereum.rs

```diff
@@ -66,8 +66,10 @@ impl EthereumAdapter {
             }
         };
 
+        let identity = ValidatorId::try_from(address.as_str())?;
```

Comment:
`address(owned ValidatorId) -> &str -> ValidatorId::try_from` Why?! Just use the address, it's an owned value.

Reply:
We've discussed this already, forgot to remove it, but you can still use `&address` because of the `Deref` impl of `String`